### PR TITLE
OpenSource : 5.26 Explain everything related to Linkerd #660

### LINKED ISSUE #660: OpenSource : 5.26 Explain everything related to `Linkerd`
### Description :

- **Explain everything related to `Linkerd`**
### Note : 

- Changes should be made inside the `Open_Source/` directory & `Opensource` branch.
-  Put  your `doc` inside `section-5`
-  Issue only for `GWOC'21` contributors.
-  Issue will be assigned on a **first come first serve basis**, **1 Issue  ==  1 PR == 1 Contributor**
-  For `Contributor` please don't create the same issue again, before creating any issue do visit [OpenSource Milestone](https://github.com/girlscript/winter-of-contributing/milestone/2)
- `Doc name` should same as the `issue title  `

## Happy Contributing 🙌

### PR DESCRIPTION
**Pull request by Khushi-Agrawal-1**
<hr>

## Description 📜

I have added documentation on the Explanation of Linkerd in Section 5 in an open-source folder of the Opensource Branch.
Fixes #660

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Open Source

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
image

<hr>

### CODE REVIEW

File: C_CPP/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Aryamanz29) [inline C_CPP/readme.md:1]: Delete this line from `C_CPP/readme.md`

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]
Delete this line from `C_CPP/readme.md`

**Comment by Khushi-Agrawal-1** [comment]
image

For me its showing that there's nothing written in that file. What should I do?

**Comment by Aryamanz29** [comment]
> image
> 
> For me its showing that there's nothing written in that file. What should I do?

Remove that blank line

**Comment by Khushi-Agrawal-1** [comment]
How do I delete the first line? The only option I can see is to delete the file.

**Comment by Aryamanz29** [comment]
> How do I delete the first line? The only option I can see is to delete the file.

Do this through your local environment, then` git add . then git commit, push`

**Comment by Khushi-Agrawal-1** [comment]
image

I deleted the line like how you told. So now its showing that I deleted something when I'm creating a PR. So is this alright?

**Comment by Khushi-Agrawal-1** [comment]
And its only showing this when I'm choosing the base as "main" and not "Opensource"

**Comment by Khushi-Agrawal-1** [comment]
image

Its showing 1 file changed when I choose "main" as base but its showing 2 files changed when I choose "Opensource". What to do?

**Comment by Khushi-Agrawal-1** [comment]
Can I use "main" as the base ?

**Comment by Aryamanz29** [comment]
> Can I use "main" as the base ?

No, Please create PR from `your_fork : Opensource Branch to girlscript : Opensource Branch`

**Pull request closed without merging**